src/test/java/uk/nhs/digital/nhsconnect/nhais/mapper/FreeTextMapperTest.java

```diff
@@ -24,6 +24,14 @@ void when_FreeTextValueExistsAndValueIsNotSet_Then_CanMap() {
         Parameters parameters = new Parameters()
             .addParameter(ParameterNames.FREE_TEXT, "");
 
+        assertThat(freeTextMapper.canMap(parameters)).isTrue();
```

Comment:
I'd expect this to be false.

Also update method name: Then_CanNotMap? threw me off initially

Reply:
I also thought like that. But `canMap()` should check only for existence of value without validating it as validation code is inside each Segment class. So if we would validate here then no error would come up later (segment would be ignored without info to end-user). For invalid data each Segment code will throw exception during mapping.

Reply:
That makes sense 👍